PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -1800,19 +1800,22 @@ void FindReplaceDlg::findAllIn(InWhat op)
 
 		_pFinder->_scintView.display();
 		_pFinder->display();
-		doSetMarkingStruct = true;
+		::SendMessage(_hParent, NPPM_DMMHIDE, 0, (LPARAM)_pFinder->getHSelf());
+		::UpdateWindow(_hParent);
+		justCreated = true;

```

Comment:
When `FindReplaceDlg::findAllIn` is called for the first time the `_pFinder` is created. On consecutive calls to the function `_pFinder` is already there and it is not `justCreated` and the code at 1812 shouldn't be executed.